packages/calcite-components/src/components/stepper/stepper.e2e.ts

```diff
@@ -262,37 +262,48 @@ describe("calcite-stepper", () => {
           <calcite-stepper-item heading="Step 3" id="step-3">
             <div>Step 3 content</div>
           </calcite-stepper-item>
+          <calcite-stepper-item heading="Step 4" id="step-4" hide-item>
```

Comment:
FIXME: `hide-item` is internal, so it shouldn't be used in tests. You'll need to update lines [like](https://github.com/Esri/calcite-design-system/blob/dev/packages/calcite-components/src/components/stepper/stepper.tsx#L321) [the](https://github.com/Esri/calcite-design-system/blob/dev/packages/calcite-components/src/components/combobox/combobox.tsx#L146-L159) [following](https://github.com/Esri/calcite-design-system/blob/dev/packages/calcite-components/src/components/table/table.tsx#L341) to use the internal prop instead. This change will ensure users can hide elements without our components making them unintentionally visible.